### PR TITLE
Core settings

### PR DESCRIPTION
**Pull request by JoelForamitti**
- Settings UI corresponding to https://github.com/MILA-Wien/collectivo/pull/120
- Logo is changed based on Core Settings
- Delete MILA specific code (will be moved to MILA repository)
- Fix small mistakes from the big restructuring of the previous PRs
- Improve loading for ObjectTable and ObjectDetail
- Improve Masonry Layout

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,7 +37,8 @@
     "vue-i18n": "^9.2.2",
     "vue-json-csv": "^2.1.0",
     "vue-router": "^4.1.5",
-    "vue-flex-waterfall": "^2.2.2"
+    "vue-flex-waterfall": "^2.2.2",
+    "@yeger/vue-masonry-wall": "^4.0.2"
```
Comment (nanu-c) [inline package.json:41]: do you really use both?

File: src/api/types.ts [hunk 2]
```diff
@@ -112,23 +112,27 @@ export interface SchemaField {
...
   label: string;
   max_length: number;
-  read_only: boolean;
+  read_only: boolean | SchemaCondition;
   required: boolean;
   write_only: boolean;
   choices?: Array<SchemaChoiceItem>;
-  condition?: SchemaCondition;
+  visible?: boolean | SchemaCondition;
+  options?: any;
```
Comment (nanu-c) [inline src/api/types.ts:120]: Must this be `any`?

**Comment by nanu-c** [comment]
Please also fix the test run

File: package.json (hunk 1 above)
Comment (JoelForamitti) [reply package.json:41]: i kept both in to make it easier to switch between branches. i have removed the old one now.

File: src/api/types.ts (hunk 2 above)
Comment (JoelForamitti) [reply src/api/types.ts:120]: I have improved the interface and made sure it is used in the main store now.

File: src/components/SidebarMenuItem.vue [hunk 3]
```diff
@@ -10,9 +13,11 @@ const props = defineProps({
 
 const item_ = JSON.parse(JSON.stringify(props.item));
 const icon = "pi pi-fw " + (props.item.icon_name ? item_.icon_name : "");
-if (item_.extension.name === "core" && item_.component === "logout") {
+if (item_.extension.name === "core" && item_.name === "logout") {
+  item_.target = "link";
   item_.link = userStore.user?.logoutUrl || "/";
 }
+console.log(item_);
```
[the remark this answers is not in the document]
Comment (JoelForamitti) [reply src/components/SidebarMenuItem.vue:20]: thanks for spotting!

**Comment by JoelForamitti** [comment]
> Please also fix the test run

done!

**Review by nanu-c — APPROVED** [review]

**Pull request merged**